src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -1,59 +1,56 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{80721425-68E5-48DC-87EA-41D63D0B45FA}</ProjectGuid>
+    <ProjectGuid>{74165807-676F-4464-B607-8770C13E8991}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>FakeItEasy</RootNamespace>
     <AssemblyName>FakeItEasy</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
OK, I eventually managed to make it work:

I still have an issue with StyleCop, because the usings now look like this in the files I had to change:

```csharp
#if !NETCORE
    extern alias mscorlib;
#endif
    using System;
    using System.Linq;
    using FluentAssertions;
    using Xunit;
#if NETCORE // this is because .NET Core doesn't support aliases (AFAIK)
    using System.Reflection;
#else
    using mscorlib::System.Reflection;
#endif
```

StyleCop complains about the order of the usings. But to make the order correct, I would have to do this, which is even more ugly:

```csharp
#if !NETCORE
    extern alias mscorlib;
#endif
    using System;
    using System.Linq;
#if NETCORE
    using System.Reflection;
#endif
    using FluentAssertions;
    using Xunit;
#if !NETCORE
    using mscorlib::System.Reflection;
#endif
```